crates/ruff_workspace/src/pyproject.rs

```diff
@@ -157,16 +168,42 @@ pub(super) fn load_options<P: AsRef<Path>>(path: P) -> Result<Options> {
         }
         Ok(ruff)
     } else {
-        let ruff = parse_ruff_toml(path);
-        if let Ok(ruff) = &ruff {
+        let mut ruff = parse_ruff_toml(&path);
```

Comment:
Is this function also called for user-level configurations? I'm asking because we should only apply this fallback to project configurations. 

We should also avoid this fallback for inherited configuration.